grml-debootstrap

```diff
@@ -1582,7 +1636,7 @@ fi
   if grep -q '^GRUB_DISABLE_LINUX_UUID=.*true' "${MNTPOINT}"/etc/default/grub 2>/dev/null ; then
     ewarn "GRUB_DISABLE_LINUX_UUID is set to true in /etc/default/grub, not adjusting root= in grub.cfg."
     ewarn "Please note that your system might NOT be able to properly boot."
-  else
+  elif [ -z "$ARM_EFI_TARGET" ]; then
```

Comment:
why do we skip this for arm64 then, is the `root=...` configuration really working as needed? *hmmm*